From 2c5b90231b2ef2933c4e6e2d4860a4ef0cd71268 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Mon, 11 Jan 2021 20:35:35 +0800
Subject: [PATCH] Revert "Revert "Revert "arm64: dts: rockchip: Temporarily
 disable its for rk3568"""

This reverts commit b14e969b5e577eabb7b246617a05d1dec642abe0.

Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
Change-Id: Ie16159d8b8e4eeac9bc39df23baf15d69126af57
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index f92ed5906cbc..66a61321de30 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -509,7 +509,6 @@
 			msi-controller;
 			#msi-cells = <1>;
 			reg = <0x0 0xfd440000 0x0 0x20000>;
-			status = "disabled";
 		};
 	};
 
-- 
2.35.3

